src/prompts/alert/index.tsx

```diff
@@ -0,0 +1,74 @@
+import './alert.scss';
+
+import { Component, h } from 'preact';
+import { PromptUiProps } from "../ui";
+
+export class Alert extends Component<PromptUiProps, never> {
+  onRequestNativePrompt = () => {
+      this.props.onPromptAccepted(this.props.config);
+  };
+
+  onRequestCancel = () => {
+      this.props.onPromptDeclined(this.props.config);
+  };
+
+  render() {
+      const classes = `kumulos-prompt kumulos-prompt-${this.props.promptManagerState} kumulos-alert-container`;
+      const config = this.props.config;
+
+      if (config.type !== 'alert') {
+          return null;
+      }
+
+      const { labels, colors } = config;
+
+      const heading = labels?.alert?.heading;
+      const body = labels?.alert?.body;
+      const iconUrl = config.iconUrl;
+
+      const cancel = labels?.alert?.declineAction ?? 'maybe later';
```

Comment:
Is it better to make some fo these things non optional?

Like if we know it's an alert, we expect the alert key to exist, and maybe we specify in config UI that the action labels are required (we can pre-fill).

Reply:
Done 👍